diff --git a/RnDiffApp/android/app/src/main/AndroidManifest.xml b/RnDiffApp/android/app/src/main/AndroidManifest.xml
index bc3a9310..f3e0d155 100644
--- a/RnDiffApp/android/app/src/main/AndroidManifest.xml
+++ b/RnDiffApp/android/app/src/main/AndroidManifest.xml
@@ -1,8 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.rndiffapp">
+  package="com.rndiffapp">

     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>

     <application
       android:name=".MainApplication"
diff --git a/RnDiffApp/ios/RnDiffApp/AppDelegate.h b/RnDiffApp/ios/RnDiffApp/AppDelegate.h
index 4b5644f2..2726d5e1 100644
--- a/RnDiffApp/ios/RnDiffApp/AppDelegate.h
+++ b/RnDiffApp/ios/RnDiffApp/AppDelegate.h
@@ -5,9 +5,10 @@
  * LICENSE file in the root directory of this source tree.
  */

+#import <React/RCTBridgeDelegate.h>
 #import <UIKit/UIKit.h>

-@interface AppDelegate : UIResponder <UIApplicationDelegate>
+@interface AppDelegate : UIResponder <UIApplicationDelegate, RCTBridgeDelegate>

 @property (nonatomic, strong) UIWindow *window;
diff --git a/RnDiffApp/android/app/src/main/java/com/rndiffapp/MainApplication.java b/RnDiffApp/android/app/src/main/java/com/rndiffapp/MainApplication.java
index bc3a9310..f3e0d155 100644
--- a/RnDiffApp/android/app/src/main/java/com/rndiffapp/MainApplication.java
+++ b/RnDiffApp/android/app/src/main/java/com/rndiffapp/MainApplication.java
