diff --git a/RnDiffApp/android/build.gradle b/RnDiffApp/android/build.gradle
index 85d8f2f8..a1e80854 100644
--- a/RnDiffApp/android/build.gradle
+++ b/RnDiffApp/android/build.gradle
@@ -9,8 +9,8 @@ buildscript {
         supportLibVersion = "27.1.1"
     }
     repositories {
-        jcenter()
         google()
+        jcenter()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.4'
@@ -23,12 +23,12 @@ buildscript {
 allprojects {
     repositories {
         mavenLocal()
+        google()
         jcenter()
         maven {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url "$rootDir/../node_modules/react-native/android"
         }
-        google()
     }
 }

diff --git a/RnDiffApp/package.json b/RnDiffApp/package.json
index 4e617645..c82829bd 100644
--- a/RnDiffApp/package.json
+++ b/RnDiffApp/package.json
@@ -7,14 +7,14 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.5.0",
-    "react-native": "0.57.0"
+    "react": "16.6.1",
+    "react-native": "0.57.7"
   },
   "devDependencies": {
     "babel-jest": "23.6.0",
     "jest": "23.6.0",
-    "metro-react-native-babel-preset": "0.47.1",
-    "react-test-renderer": "16.5.0"
+    "metro-react-native-babel-preset": "0.49.2",
+    "react-test-renderer": "16.6.1"
   },
   "jest": {
     "preset": "react-native"
